src/ChatFeed/ChatHeader/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { Props } from './props';
+import { styles } from './styles';
+
+import { Row, Col, setConfiguration } from 'react-grid-system';
+
+setConfiguration({ maxScreenClass: 'xl', gutterWidth: 0 });
+
+export const ChatHeader: React.FC<Props> = ({
+  id = '',
+  title = '',
+  description = '',
+  style = {},
+}) => {
+  return (
+    <Row
+      className="ce-chat-title"
+      style={{ ...styles.titleSection, ...style.titleSection }}
+    >
+      <Col
+        xs={2}
+        sm={0}
+        style={{
+          ...styles.mobileOptiom,
+          ...{ left: '6px' },
```

Comment:
could we just put `...{ left: '6px' },` in the default styles?

Reply:
Yes! Adding

Reply:
Actually no. `mobileOptiom` is used on the left and right of the page and need this positioning different for each

- Default
- Usage Specific
- Prop Override

Its still in line with that